internal/command/lfsc/lfsc.go

```diff
@@ -0,0 +1,122 @@
+// Package lfsc implements the LiteFS Cloud command chain.
+package lfsc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/gql"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/lfsc-go"
+)
+
+const (
+	tokenExpiry = 1 * time.Minute
+)
+
+// New initializes and returns a new apps Command.
+func New() *cobra.Command {
+	const (
+		long = `Commands for managing LiteFS Cloud databases`
+
+		short = "LiteFS Cloud management commands"
+
+		usage = "litefs-cloud <command>"
+	)
+
+	cmd := command.New("usage", short, long, nil)
+	cmd.Aliases = []string{"litefs-cloud", "lfsc"}
```

Comment:
ah.. this was what my first comment was about. +1 to `litefs-cloud` but it doesn't need to be aliases. It is extracted from the usage string.